htdocs/user/group/ldap.php

```diff
@@ -75,6 +75,12 @@
 		$dn = $object->_load_ldap_dn($info);
 		$olddn = $dn; // We can say that old dn = dn as we force synchro
 
+		if ($ldap->serverType == "activedirectory") {
+			$info['sAMAccountName'] = $object->name;
+		}
+
+		unset($info['member']);
```

Comment:
Are you sure we must unset this all the times or only if ($ldap->serverType == "activedirectory") {

Reply:
The line was not intended to be in this pull request.
Removed it. 
Thanks for the review and sorry for the time loss.